modules/knative-serving-advanced-config.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+// * /serverless/installing_serverless/serverless-install-config-options.adoc
+
+[id="knative-serving-advanced-config_{context}"]
+= Knative Serving supported installation configuration options
+
+This guide provides information for cluster administrators about advanced installation configuration options for Knative Serving.
+
+[IMPORTANT]
+====
+Do not modify any YAML contained inside the `config` field. Some of the configuration values in this field are injected by the {ServerlessOperatorName}, and modifying them will cause your deployment to become unsupported.
+====
+
+image::serving-form-view.png[Create Knative Serving form in web console Administrator Perspective]
+
+== Controller Custom Certs
+
+If your registry uses a self-signed certificate, you must enable tag-to-digest resolution by creating a ConfigMap or Secret.
+The {ServerlessOperatorName} then automatically configures Knative Serving controller access to the registry.
+
+To enable tag-to-digest resolution, the Knative Serving controller requires access to the container registry.
+
+[IMPORTANT]
+====
+The ConfigMap or Secret must reside in the same namespace as the Knative Serving custom resource definition.
+====
+
+The following example triggers the {ServerlessOperatorName} to:
+
+. Create and mount a volume containing the certificate in the controller.
+. Set the required environment variable properly.
+
+.Example YAML
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  controller-custom-certs:
+    name: certs
+    type: ConfigMap
+----
+
+The following example uses a certificate in a ConfigMap named `certs` in the `knative-serving` namespace.
+
+You can also configure controller custom certs using the form view.
```

Comment:
In general throughout, wondering if certs can be spelled out when not referring to a field or object name.